### DISCUSSION TITLE
How do I see what versions of redis are supported for jedis client version 2.6.0?

### DISCUSSION THREAD

**Discussion by cfoskin** [xetorthio/jedis#3178] [Q&A] [answered]
We have jedis version 2.6.0 in a legacy system and need to see what versions of Redis it supports (ideally what is the latest version it supports). Is there some docs or a matrix with the compatibility?

**Comment by sazzad16** [comment, marked as answer]
There is no guaranteed compatibility table, but Jedis 2.6.0 is likely to support even the latest Redis.

**Reply by cfoskin** [reply]
Thank you 👍
